sydent/http/httpserver.py

```diff
@@ -92,6 +93,7 @@ def __init__(self, sydent: "Sydent", lookup_pepper: str) -> None:
         matrix.putChild(b"identity", identity)
         identity.putChild(b"api", api)
         identity.putChild(b"v2", v2)
+        identity.putChild(b"versions", VersionsServlet())
```

Comment:
a test for this endpoint wouldn't go amiss...

Reply:
Adding one to the IS tester likely makes the most sense.

Reply:
See matrix-org/matrix-is-tester#25